lib/auth/auth_with_roles.go

```diff
@@ -1567,6 +1573,7 @@ func (a *ServerWithRoles) generateUserCerts(ctx context.Context, req proto.UserC
 		return nil, trace.Wrap(err)
 	}
 
+	// TODO(zmb3): add CA certs?
```

Comment:
Reviewers - what do you think? Any harm in including the CA certs when we generate user certs just so it behaves similarly to when we generate host certs?

Reply:
Sounds alright.